bin/20newsgroup-replication.py

```diff
@@ -0,0 +1,46 @@
+import sys
```

Comment:
hrm, I'm wondering if we should have a separate `scripts/` directory and save `bin/` just for simple shell scripts?

Reply:
I can do that.